thsvs_simpler_vulkan_synchronization.h

```diff
@@ -182,9 +182,10 @@ typedef enum ThsvsAccessType {
     THSVS_ACCESS_NONE,                                                      // No access. Useful primarily for initialization
 
 // Read access
-    // Requires VK_NVX_device_generated_commands to be enabled
-    THSVS_ACCESS_COMMAND_BUFFER_READ_NVX,                                   // Command buffer read operation as defined by NVX_device_generated_commands
-
+#if defined(VK_NV_device_generated_commands) || defined(VK_NVX_device_generated_commands)
+    // Requires VK_NV_device_generated_commands to be enabled
+    THSVS_ACCESS_COMMAND_BUFFER_READ_NV,                                    // Command buffer read operation as defined by NV_device_generated_commands
+#endif

```

Comment:
Why are there ifdefs around this? I don't ifdef anything else in here. I could agree to ifdeffing all of the extensions but just this one seems out of place.

Reply:
The goal was to continue to support older SDKs that expose the NVX extension but not the NV variant. If that's not a priority, I'd be totally fine just replacing NVX->NV unconditionally.